confd/tests/mock_data/calicoctl/mesh/static-routes/kubectl-input.yaml

```diff
@@ -123,18 +132,21 @@ spec:
       targetPort: 80
 
 ---
-kind: Endpoints
-apiVersion: v1
+kind: EndpointSlice
```

Comment:
I think it's probably worth augmenting one of these tests to deploy multiple EndpointSlices, to make sure our logic handles that properly.